app/lib/link.rb

```diff
@@ -0,0 +1,16 @@
+class Link
+
+  def initialize from_mac, to_mac, logic
+    @from_board = Board.find_by mac: from_mac
+    @to_board = Board.find_by mac: to_mac
+    @logic = logic
+  end
+
+  def run
+    if @logic == "add"
```

Comment:
Ca sa stii, poti defini sus un `attr_reader :logic` care iti va crea automat un getter si in run ai putea sa folosesti `if logic == 'add'` in loc sa accesezi direct class variable 

https://www.codecademy.com/en/forum_questions/50f0192b102455349200372d